### PR TITLE
36 add docker compose

### LINKED ISSUE #36: Add Docker Compose
Update:
 - the service contexts need to point to the new path of the services
 - update the env variables
 - add the missing services (mailer, reporting)

### PR DESCRIPTION
**Pull request by Tamas99**

### CODE REVIEW

File: compose.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  profile:
+    build:
+      context: backend/profile
+    image: profile:0.1
+    container_name: profile
+    ports:
+      - 8010:8010
+    environment:
+      PORT: 8010
+      MONGODB_URI: mongodb+srv://ozddev:GzNZ3FzNgy8hNmMi@zemcluster1.dz4rzdu.mongodb.net/employee?retryWrites=true&w=majority
```
Comment (Galterius) [inline compose.yml]: I don't know if its possible or not, but can you somehow take this string from the .env so its not exposed? Or maybe add the local connection string as default `mongodb://localhost:27017`

Comment (Galterius) [reply compose.yml]: Also this repo is public, and other can see this connection string now

Comment (Galterius) [reply compose.yml]: Also you can use this official image for a new local mongo docker compose service https://hub.docker.com/_/mongo

**Review by Galterius — CHANGES_REQUESTED** [review]

File: compose.yml (hunk 1 above)
Comment (Tamas99) [reply compose.yml]: I have changed it to localhost for now

**Review by Galterius — DISMISSED** [review]

**Review by Galterius — APPROVED** [review]

**Review by Galterius — DISMISSED** [review]

**Pull request merged**